examples/session/session-reservation.py

```diff
@@ -80,7 +80,7 @@
         )
     )
     vi = init_with_options_response.vi
-    print(f"Session initialized with name {session_name} and id {vi.id}.\n")
+    print(f"Session initialized with name {session_name}.\n")
```

Comment:
I know why this change is needed, given Chris' recent submission. I have no idea how his PR got through without including this. FYI @astarche @reckenro

Reply:
Agreed, this should've failed the `validate_python` step in Chris' PR or at least his merge to main. I created a Tech Debt [AB#2219712](https://dev.azure.com/ni/DevCentral/_workitems/edit/2219712) item to investigate and hopefully figure out / fix what's going on here.